pkg/util/version/const.go

```diff
@@ -84,3 +84,8 @@ func MdsdImage(acrDomain string) string {
 
 	return acrDomain + "/genevamdsd:master_20211223.1"
 }
+
+// MUOImage contains the location of the Managed Upgrade Operator container image
+func MUOImage(acrDomain string) string {
+	return acrDomain + "/managed-upgrade-operator:aro-b1"
```

Comment:
This is a current yolo tag, since it's built with changes that have since been upstreamed. I felt like using the image we'd already deployed and know works was a lower risk than building what will end up being the future one (which will be a tag matching the commit hash of the MUO repo, or MUO tag).